extensions/desktop/common/desktop-exports

```diff
@@ -391,7 +391,7 @@ append_dir LD_LIBRARY_PATH "$SNAP/testability/$ARCH/mesa"
 # Gdk-pixbuf loaders
 export GDK_PIXBUF_MODULE_FILE="$XDG_CACHE_HOME/gdk-pixbuf-loaders.cache"
 export GDK_PIXBUF_MODULEDIR="$SNAP_DESKTOP_RUNTIME/usr/lib/$ARCH/gdk-pixbuf-2.0/2.10.0/loaders"
-if [ "$needs_update" = true ]; then
+if [ "$needs_update" = true ] || [ ! -f $GDK_PIXBUF_MODULE_FILE ]; then
```

Comment:
```suggestion
if [ "$needs_update" = true ] || [ ! -f "$GDK_PIXBUF_MODULE_FILE" ]; then
```

```
In ./extensions/desktop/common/desktop-exports line 394:

if [ "$needs_update" = true ] || [ ! -f $GDK_PIXBUF_MODULE_FILE ]; then

                                        ^---------------------^ SC2086: Double quote to prevent globbing and word splitting.
```

Reply:
Done!